posthog/api/survey.py

```diff
@@ -751,6 +773,249 @@ def responses_count(self, request: request.Request, **kwargs):
 
         return Response(counts)
 
+    def _validate_and_parse_dates(
```

Comment:
@ioannisj @marandaneto now handling both parsing and validation on this function